core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -155,11 +157,17 @@ angular.module('oppia').controller(
           return;
         }
 
-        [$scope.activeSuggestionId, $scope.activeSuggestion] = (
-          remainingSuggestions.pop());
+        [$scope.activeSuggestionId, $scope.activeContribution] = (
+          remainingContributions.pop());
+        $scope.activeSuggestion = $scope.activeContribution.suggestion;
+        $scope.activeSuggestionDetails = $scope.activeContribution.details;
         ContextService.setCustomEntityContext(
           IMAGE_CONTEXT.EXPLORATION_SUGGESTIONS,
           $scope.activeSuggestion.target_id);
+        var details = $scope.activeSuggestionDetails;
```

Comment:
This isn't really necessary, right? Why not just use `$scope.activeSuggestionDetails`?

Reply:
done